lib/auth/password_test.go

```diff
@@ -294,6 +295,50 @@ func TestServer_ChangePassword(t *testing.T) {
 	}
 }
 
+// This test asserts that an attacker is unable to change password without
+// providing the old one if they take over a user's web session and use a
+// different type of WebAuthn challenge that would be normally requested by the
+// Web UI. This is a regression test for
+// https://github.com/gravitational/teleport-private/issues/1369.
+func TestServer_ChangePassword_FailsWithoutOldPassword(t *testing.T) {
+	t.Parallel()
+
+	srv := newTestTLSServer(t)
+	mfa := configureForMFA(t, srv)
+	authServer := srv.Auth()
+	ctx := context.Background()
+
+	username := mfa.User
+	newPass := []byte("capybarasarecool123")
+
+	clt, err := srv.NewClient(TestUser(username))
```

Comment:
nit: spell it out, say what client it is. There are too many possible "clients" we can have, it gets confusing quickly.

```suggestion
	userClient, err := srv.NewClient(TestUser(username))
```

(A lot of the older code does `srv` and `clt`, so I see where you are coming from.)